# marker_pose.srv
bool success 
# Request
---
geometry_msgs/PoseStamped pose_msg


